dlib/cmake_utils/find_blas.cmake

```diff
@@ -19,8 +19,11 @@
 #  openmp_libraries         - Set to Intel's OpenMP library if and only if we
 #                             find the MKL.
 
+INCLUDE(CheckFunctionExists)
```

Comment:
Remove all the other places that include CheckFunctionExists if this is going to live here.

Reply:
So I just remove this line? Because other places have `CheckFunctionExists` ?